launch/client.py

```diff
@@ -1187,7 +1185,6 @@ def _async_request(
                 `abcabcab-cabc-abca-0123456789ab`
         """
         validate_task_request(url=url, args=args)
-        endpoint = self.get_model_endpoint(endpoint_name)
```

Comment:
is this just a postgres call?

Reply:
Yes, but also Redis to get some cached data about the infra state (and k8s api server if Redis cache doesn't contain the info)